examples/AspNetCore/prometheus.yml

```diff
@@ -0,0 +1,26 @@
+# my global config
+global:
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+  # Attach these labels to any time series or alerts when communicating with
+  # external systems (federation, remote storage, Alertmanager).
+  external_labels:
+      monitor: 'codelab-monitor'
+
+# Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+rule_files:
+  # - "first.rules"
+  # - "second.rules"
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
```

Comment:
This line is misleading, consider removing it.

General question - do we intend to have a minimal config file? E.g.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/71a6969c1c665642f8fec7afa16fc00b436890a8/examples/Console/TestPrometheusExporter.cs#L39-L49

Reply:
The intent is to provide a very minimal configuration, I was just trying to do the bare-minimum to set up scraping since prometheus is different from other exporters in that way and doesn't "just work". I based my configuration off of https://docs.docker.com/config/daemon/prometheus/, and didn't really look at the comments they put in their example. Once I had linting issues, though, because of some whitespace character, I looked at them and decided the comments weren't necessary, since the point of this project isn't to teach people how to configure prometheus.

Now that I've removed the comments, does this file look good to you?

Reply:
Yep, it looks great! Thanks! 👍